src/azure-cli/az.bat

```diff
@@ -1,11 +1,11 @@
 @echo off
 setlocal
 
-SET PYTHONPATH=%~dp0\src;%PYTHONPATH%
+SET PYTHONPATH=%~dp0\..;%PYTHONPATH%
```

Comment:
`src/azure-cli/az.bat` and `src/azure-cli/az` are used in pip installation. I think it's not necessary to change `PYTHONPATH`.

Reply:
Done